docs/library/common/ad_dds/index.rst

```diff
@@ -0,0 +1,331 @@
+amplitude converters have dedicated sub-modules, as described above. When the
+frequency is changed by software the 4 phases accumulator are align for the new
+frequency word and/or frequency phase shift. The counter increment value will be
+multiplied with clock ratio(4), to get a continuity of the 4 consecutive samples
+generated at t, t+1, t+2, and so on.
+
+.. image:: dds_clk_ratio_4.svg
+
+For scenarios where the synchronization signal comes from an external source and
+it is high for a longer period of time, the phase accumulator stages will be
+hold in reset, in order to avoid a noise like signal, caused by sending all the
+summed outputs of each DDS stage.
+
+There is a minimum synchronization pulse width(delay) of n clock cycles, that is
+required to synchronize all phase accumulator stages, where n is equal to the
+CLK_RATIO.
+
+.. image:: fw_sync_basics.svg
+
+.. wavedrom::
+
+   {
+     "signal" : [
+       { "name": "clk",        "wave": "P....................."},
+       { "name": "external sync","wave": "0.1..|..0............."},
+       { "name": "phase init",  "wave": "0.......1...0........."},
+       { "name": "tone 1 gen0",  "wave": "=.......4=..44|4444444","data":["","i0","","i0+","i0+","i0+","..."]},
+       { "name": "tone 1 gen1",  "wave": "=........4=.44|4444444","data":["","i1","","i1+","i1+","i1+","..."]},
+       { "name": "tone 1 gen2",  "wave": "=.........4=44|4444444","data":["","i2","","i2+","i2+","i2+","..."]},
+       { "name": "tone 1 gen3",  "wave": "=..........444|4444444","data":["","i3","i3+","i3+","i3+","..."]},
+       { "name": "tone 1",     "wave": "=.......======|4444444","data":["","0","0","0","0","0","...","s0-3","s4-8","s8-C","..."]},
+       { "name": "tone 2 gen0",  "wave": "=.......5=..55|5555555","data":["","i0","","i0+","i0+","i0+","..."]},
+       { "name": "tone 2 gen1",  "wave": "=........5=.55|5555555","data":["","i1","","i1+","i1+","i1+","..."]},
+       { "name": "tone 2 gen2",  "wave": "=.........5=55|5555555","data":["","i2","","i2+","i2+","i2+","..."]},
+       { "name": "tone 2 gen3",  "wave": "=..........555|5555555","data":["","i3","i3+","i3+","i3+","..."]},
+       { "name": "tone 2",     "wave": "=.......======|5555555","data":["","0","0","0","0","0","...","s0-3","s4-8","s8-C","..."]},
+       ],
+     foot: {
+       text: ['tspan', 'Frequency word sync at CLK_RATIO=4'],
+     }
+   }
+
+
+In the above diagram example:
+
+* CLOCK_RATIO = 4
+* i0 = phase offset
+* i1 = i0 + FW
+* i2 = i1 + FW
+* i3 = i2 + FW
+
+Each "i" is on 16 bits and each "s"(0-3) is on 64 bits.
+i(t)+ is the value of the previous i(t) plus the increment(FW*CLOCK_RATIO).
+
+It should be mentioned that after the phase init fall-edge, until the first
+valid sample, there is a delay caused by the phase to angle converter type and
+in the case of the CORDIC type, number of rotation stages will also have a
+direct impact on this clock period delay.
+
+Performance analysis
```

Comment:
I think it would be better if we don't mention this, since we have done measurements so far.